Deploy/kubernetes/templates/jobservice.cm.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: harbor-jobservice-config
+data:
+  MYSQL_HOST: mysql
+  MYSQL_PORT: "3306"
+  MYSQL_USR: root
+  MYSQL_PWD: "##db_password##"
```

Comment:
nit:  I believe it's more common to use `{{` or `{%` as placeholder instead of `##`.  `##` is error prone: it may be treated as comment.